vaadin-notification-flow-parent/vaadin-notification-flow-integration-tests/src/main/java/com/vaadin/flow/component/notification/tests/NotificationView.java

```diff
@@ -15,73 +15,62 @@
 import com.vaadin.flow.component.html.Div;
-import com.vaadin.flow.component.html.H2;
 import com.vaadin.flow.component.html.NativeButton;
 import com.vaadin.flow.component.html.Span;
 import com.vaadin.flow.component.notification.Notification;
 import com.vaadin.flow.component.notification.Notification.Position;
-import com.vaadin.flow.component.notification.NotificationVariant;
-import com.vaadin.flow.component.orderedlayout.VerticalLayout;
 import com.vaadin.flow.router.Route;
-import com.vaadin.flow.server.StreamRegistration;
-import com.vaadin.flow.server.StreamResource;
 
-/**
- * View for {@link Notification} demo.
- */
+@CssImport("./notification-not-animated-styles.css")
 @Route("vaadin-notification")
 public class NotificationView extends Div {
 
-    private static final String BUTTON_CAPTION = "Open notification";
-
     public NotificationView() {
-        createDefaultNotificaiton();
+        createDefaultNotification();
         createNotificationWithPosition();
         createNotificationUsingStaticConvenienceMethod();
         createNotificationWithComponents();
-        addStyledNotificationContent();
-        createThemeVariants();
     }
 
-    private void createDefaultNotificaiton() {
-        NativeButton button = new NativeButton(BUTTON_CAPTION);
+    private void createDefaultNotification() {
+        NativeButton button = new NativeButton("Default notification");
         Notification notification = new Notification(
                 "This notification has text content", 3000);
         button.addClickListener(event -> notification.open());
         button.setId("default-notification-button");
         notification.setId("default-notification");
-        addCard("Default Notification", button);
+        add(button);
     }
 
     private void createNotificationWithPosition() {
-        NativeButton button = new NativeButton(BUTTON_CAPTION);
+        NativeButton button = new NativeButton("Notification with position");
         Notification notification = new Notification(
                 new Span("This notification is located on Top-Left"));
         notification.setDuration(3000);
         notification.setPosition(Position.TOP_START);
         button.addClickListener(event -> notification.open());
         button.setId("position-notification-button");
         notification.setId("position-notification");
-        addCard("Notification with position", button);
+        add(button);
     }
 
     private void createNotificationUsingStaticConvenienceMethod() {
-        Notification notification = Notification.show(
-                "This is a notification created with static convenience method");
-        notification.setId("static-notification");
-        addCard("Notification using static convenience method");
+        NativeButton button = new NativeButton(
+                "Notification using static method");
+        button.addClickListener(event -> {
+            Notification notification = Notification.show(
```

Comment:
This notifications was previously opening immediately when opening the page. 
Changed it to use dedicated `<button>` with a click listener to align with others.